### ISSUE TITLE
List-like lines should be escaped

### ISSUE THREAD

**Issue by marijnvdwerf** [thephpleague/html-to-markdown#71]
**Input**:
```html
<p>
+ Siri works well for TV and movies<br>
+ Really fast<br>
+ Games are a fun addition<br>
- Siri is extremely limited<br>
- Have to log in to every app individually<br>
- No 4K support
</p>
```

**Actual**:
```md
+ Siri works well for TV and movies  
+ Really fast  
+ Games are a fun addition  
- Siri is extremely limited  
- Have to log in to every app individually  
- No 4K support
```

**Expected**:
```md
\+ Siri works well for TV and movies  
\+ Really fast  
\+ Games are a fun addition  
\- Siri is extremely limited  
\- Have to log in to every app individually  
\- No 4K support
```

**Comment by colinodell** [comment]
@andreskrey Would you be interested in tackling this one as well? ;-)

**Comment by andreskrey** [comment]
Done! #107

**Comment by colinodell** [comment]
YTMND! Thanks so much :)

Fixed via #107.
